README.md

```diff
@@ -36,6 +36,7 @@ url = await run("itemid", [... other arguments])
 | portal_url     | str  | The URL of the ArcGIS Portal instance to use. Defaults to ArcGIS Online: `"https://www.arcgis.com"`                                                                                                                            |
 | token          | str  | The Portal access token to be used to access secured resources. If not provided requests to secured resources will fail.                                                                                                       |
 | culture        | str  | The culture to use for localization. For example `"en-US"`.                                                                                                                                                                    |
+| result_file_name    | str  | The desired name of the output file.  |
```

Comment:
Do you mind updating the docs here and in code to what you came up with on the JS side?

Reply:
I've made it similar, but omitted the part about the browser tab as that is not likely relevant.